rpcs3/Emu/NP/np_handler.h

```diff
@@ -415,6 +418,12 @@ namespace np
 
 		ticket current_ticket;
 
+		// Clan ticket
+		std::mutex mutex_clan_ticket;
```

Comment:
I feel you could replace all this with one `atomic_t<u32> ticket_ready = 0`. You do not really need a complex system here with a mutex to protect the ticket as there should only be 2 states, ticket available or ticket not available and once the ticket is available it shouldn't change ever afaik?